api/proto/config/deck.proto

```diff
@@ -14,6 +14,9 @@ import "notification/slack.proto";
 import "notification/twilio.proto";
 
 message Deck {
+  reserved 6;
```

Comment:
(Happy to change this to however we decide to handle pre-GA field removals, per https://github.com/spinnaker/kleat/pull/129#discussion_r465915628)

Reply:
Removed reserved statement & updated field numbering.